threeML/plugins/OGIPLike.py

```diff
@@ -56,7 +53,7 @@ def __init__(self, name, pha_file, bak_file=None, rsp_file=None, arf_file=None,
 
         # Get a PHA instance with the background
 
-        self._bak = self._get_pha_instance(bak_file, file_type='background')
+        self._bak = self._get_pha_instance(bak_file, file_type='background', spectrum_number=spectrum_number)
```

Comment:
originally the spectrum number was specified in the file name, as "my_spectrum.pha{1}". Is that still supported?